ui/cypress/e2e/dashboardsView.test.ts

```diff
@@ -53,4 +59,89 @@ describe('Dashboard', () => {
     cy.getByTestID('save-cell--button').click()
     cy.getByTestID('cell--view-empty').should('have.length', 1)
   })
+
+  const getSelectedVariable = (contextID: string, variableID: string) => win =>
+    win.store.getState().variables.values[contextID].values[variableID]
+      .selectedValue
+
+  it('can manage variable state with a lot of pointing and clicking', () => {
+    cy.get<Organization>('@org').then(({id: orgID}) => {
```

Comment:
@Asalem1 I noticed you changing to
```js
cy.get<Organization>('@org').then(({id: orgID})
``` 
to 
```js
cy.get('@org').then(({id: orgID}: Organization)
```
in some recent PRs - is there a preferred way of doing this?

Reply:
@hoorayimhelping, @121watts had me do it on my first PR last week and my linter has been yelling at me to change them on files that I visit so I've just been following that. From my perspective it lays out a clearer understanding of what the expectations of the parameter will be by setting it next to the parameter as opposed to the `get`